clients/input_copy.go

```diff
@@ -93,6 +93,21 @@ func (s *InputCopy) CopyInputToS3(requestID string, inputFile *url.URL, decrypto
 	return
 }
 
+func getSignedURL(osTransferURL *url.URL) (string, error) {
+	// check if plain https is accessible, if not then the bucket must be private and we need to generate a signed url
```

Comment:
@victorges @thomshutt is this too hacky, what do you think?

Reply:
I quite like this, maybe just add to the comment that signed URLs don't work with Mediaconvert for whoever comes along in a year and thinks "Why don't we just sign everything?"